ospfd/ospf_sr.h

```diff
@@ -61,7 +61,7 @@
 
 /* SID/Label Sub TLV - section 2.1 */
 #define SUBTLV_SID_LABEL		1
-#define SUBTLV_SID_LABEL_SIZE		8
+#define SUBTLV_SID_LABEL_SIZE		4
```

Comment:
Even if the correction is right, it is not sufficient and will break SRGB and SRLB subTLV size computation in ospf_ri.c at line 451 and 475.

So, you need to also correct in ospf_ri.c the SRGB and SRLB subTLV size computation e.g. by using a new define like the one I introduced in my PR #8408 i.e. `#define RI_SR_TLV_SRLB_LABEL_RANGE_SIZE 12` and then using it at line 450 and 474 of ospf_ri.c

Reply:
Thank you! I pushed the changes. Let's keep this PR because it's fixing a separate problem. I put the same line for RI_SR_TLV_LABEL_RANGE_SIZE as you did, so our PRs shouldn't conflict with each other.